### PR TITLE
Fixes showing no notification when initiating a payment

### LINKED ISSUE #184: `Payment in flight` notification does not show up when initiating a payment
## Describe the bug
Sometimes the `Payment in flight` notification does not immediately show up, when initiating a payment. 

## Expected behavior
The notification should show up as a feeback to the user. The `pending_transfers` endpoint  should be called after closing the payment dialog.

## System
- Raiden: v0.200.0rc1
- WebUI: v0.10.1
- Environment: development
- Network: Görli
- User agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:70.0) Gecko/20100101 Firefox/70.0

### PR DESCRIPTION
**Pull request by manuelwedler**
Fixes #184

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -71,20 +71,21 @@ export class AppComponent implements OnInit, OnDestroy {
...
-            () => {
-                this.notificationBlink = 'rgba(255, 255, 255, 0.5)';
-                setTimeout(() => {
+        const newNotificationSubscription = this.notificationService.newNotification$
+            .pipe(
+                tap(() => {
+                    this.notificationBlink = 'rgba(255, 255, 255, 0.5)';
+                }),
+                delay(150),
+                tap(() => {
```
Comment (andrevmatos) [inline src/app/app.component.ts]: Seems the _of_ tap is being reset 150ms after the start, regardless of a new notification coming, which can give issues if multiple notifications come up in the mean time. May I suggest a `switchMap` which _starts_ at subscription and then returns a `timer` observable to reset it but which will be unsubscribed if a new notification comes?

Comment (andrevmatos) [reply src/app/app.component.ts]: Something like:
```
const newNotificationSubscription = this.notificationService.newNotification$
  .pipe(
    switchMap(() => ((this.notificationBlink = 'rgba...'), timer(150))),
  ).subscribe(() => (this.notificationBlink = 'black'));

Comment (andrevmatos) [reply src/app/app.component.ts]: Or even better with `debounceTime`:
```
const newNotificationSubscription = this.notificationService.newNotification$
  .pipe(
    tap(() => (this.notificationBlink = 'rgba...')),
    debounceTime(150),
  ).subscribe(() => (this.notificationBlink = 'black'));
```

Comment (manuelwedler) [reply src/app/app.component.ts]: That's way cleaner. I'll go with `debounceTime`. Thanks a lot!

**Pull request merged**